src/actions/platform-status-action.js

```diff
@@ -0,0 +1,9 @@
+import SwaggerManagerApi from "../services/api-client/swagger-manager-api";
+export const UPDATE_PLATFORM_STATUS = "UPDATE_PLATFORM_STATUS";
+
+export function updatePlatformStatus() {
```

Comment:
I think it will be better to create more general platform-actions function where we will keep all related to platform actions